wireapp/wire-android-reloaded
# feat: add audio permission check for when receiving an incoming call

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

When receiving an incoming call and the user hasn't given any permissions and accepted the call, the app would crash because it tried to get audio but there is no permission.

### Solutions

Ask for user permission to use its audio when receiving an incoming call.
If user gives permission -> accepts the call (if it is the first time)
If user denies permission -> rejects the call

#### How to Test

- Open the app
- send a message to the caller
- receive an incoming call
- tap accept call
- permission prompt should pop up
